packages/server-core/src/projects/project-builder-tags/project-builder-tags.test.ts

```diff
@@ -70,12 +73,15 @@ describe('project-builder-tags.test', () => {
 
     testUserApiKey = await app.service(userApiKeyPath).create({ userId: testUser.id })
 
-    await app.service(identityProviderPath).create(
+    await app.service(identityProviderPath)._create(
```

Comment:
You can use `identityProviderDataResolver.resolve()` method here to populate default values.